tests/gtest/avifanimationtest.cc

```diff
@@ -23,6 +23,7 @@ TEST(AvifDecodeTest, AnimatedImage) {
             AVIF_RESULT_OK);
   ASSERT_EQ(avifDecoderParse(decoder.get()), AVIF_RESULT_OK);
   EXPECT_EQ(decoder->alphaPresent, AVIF_FALSE);
+  EXPECT_EQ(decoder->animationTrackPresent, AVIF_TRUE);
```

Comment:
Please add a variant of this test with

```
  ASSERT_EQ(avifDecoderSetSource(decoder.get(),
                                 AVIF_DECODER_SOURCE_PRIMARY_ITEM),
            AVIF_RESULT_OK);
```

We should have in this variant of the test:

```
  EXPECT_EQ(decoder->imageCount, 1);
```

You might want to change the for loop at the end of the test to compare `i < decoder->imageCount` instead of `i < 5`.

Reply:
Done. I added a separate test.